plugins/PrivacyManager/lang/en.json

```diff
@@ -83,6 +83,9 @@
         "DoNotTrack_SupportDNTPreference": "Support Do Not Track preference",
         "EstimatedDBSizeAfterPurge": "Estimated database size after purge",
         "EstimatedSpaceSaved": "Estimated space saved",
+        "ForceCookielessTracking": "Force tracking without cookies",
+        "ForceCookielessTrackingDescription": "Enabling this option will automatically update %1$s, so it contains some additional code to ensure all trackers won't use cookies. Additionally Matomo will ignore all tracking cookies on server side.",
```

Comment:
Could maybe say `Enabling this option will automatically update the JavaScript tracker %1$s to ensure cookies will be disabled. Additionally Matomo will ignore all tracking cookies on server side.` Just shortened it bit and making it bit more clear this affects only JS tracker but not other SDKs

Reply:
we should also add a note: `Cookies will be disabled even when using the consent methods in Matomo tracker and calling for example the cookie consent methods won't enable cookies`.